app/utilities/schema.py

```diff
@@ -177,7 +178,44 @@ def load_schema_from_url(schema_url, language_code):
 
     constructed_schema_url = f"{schema_url}?language={language_code}"
 
-    req = requests.get(constructed_schema_url)
+    session = requests.Session()
+
+    retries = Retry(
```

Comment:
I think the total retries, the status codes to retry, the max backoff and the timeout should be module level constants.

Reply:
Added all these, I am not sure do we want to keep ALLOWED_METHODS as a set with `GET` only?

Reply:
Ok, removed allowed methods, thanks!